ftplugin/javascript/cfi.vim

```diff
@@ -156,13 +177,15 @@ function! s:finder.get_func_name() "{{{
         \ '\(function\)\s*\(\*\)\=\s*',
         \ '\1\2', '')
 
-  return (g:cfi_javascript_show_assignment ? l:assignment . ': ' : '')
-        \. l:variable_name
-        \. (!empty(l:function_type) ? l:function_type . ' ' : '')
-        \. l:function_name
-        \. l:function_arguments
-        \. l:function_body
-
+  " Use the setting dict to selectively show parts of the name
+  " Don't let the user break this, cycle through our dictionary for keys
+  let l:result = ''
+  for var in keys(s:show_defaults)
```

Comment:
It's just a for loop if the variables are taken directly. I'll change then where they are defined.

I used `s:show_defaults` for future compatibility. If you add or remove keys from it, the plugin continues to work, even if the user defines the dictionary directly on their vimrc.